src/Deployment/RoslynDeployment.csproj

```diff
@@ -38,14 +45,34 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
       <IncludeOutputGroupsInVSIX>VSIXContainerProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
     </ProjectReference>
-    <ProjectReference Include="..\VisualStudio\VisualStudioInteractiveComponents\Roslyn.VisualStudio.InteractiveComponents.csproj">
-      <Name>VisualStudioInteractiveComponents</Name>
-      <VSIXSubPath>Vsixes</VSIXSubPath>
-      <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
-      <IncludeOutputGroupsInVSIX>VSIXContainerProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
-    </ProjectReference>
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.VisualStudio.SDK.Analyzers" Version="$(MicrosoftVisualStudioSDKAnalyzersVersion)" PrivateAssets="all" />
   </ItemGroup>
+  <Target Name="UpdateFeed" BeforeTargets="PrepareForBuild" Condition="'$(GitBranchName)' != ''" Inputs="UpdateFeed.pkgdef" Outputs="$(IntermediateOutputPath)\UpdateFeed.pkgdef">
+    <PropertyGroup>
+      <UpdateFeedContent>$([System.IO.File]::ReadAllText('$(MSBuildProjectDirectory)\UpdateFeed.pkgdef'))</UpdateFeedContent>
+      <UpdateFeedContent>$(UpdateFeedContent.Replace('$feature$', '$(GitBranchName)'))</UpdateFeedContent>
+    </PropertyGroup>
+    <WriteLinesToFile File="$(IntermediateOutputPath)\UpdateFeed.pkgdef"
+                      Lines="$(UpdateFeedContent)" 
+                      Overwrite="true"
+                      WriteOnlyWhenDifferent="true" />
+    <ItemGroup>
+      <Content Include="$(IntermediateOutputPath)\UpdateFeed.pkgdef" 
+               Link="UpdateFeed.pkgdef"
+               CopyToOutputDirectory="PreserveNewest" 
+               IncludeInVSIX="true" />
+    </ItemGroup>
+  </Target>
```

Comment:
❗ This doesn't tie in with the necessary up-to-date checks for MSBuild and the IDE